src/test/resources/features_integration/functionality/example_flows.feature

```diff
@@ -5,9 +5,7 @@ Feature: sample karate test script
     * url 'https://jsonplaceholder.typicode.com'
 
   Scenario: get all users and then get the first user by id
+    * def user_by_id = call read('../features_unitary/functionality/example_unitary.feature@user_by_id_params') { userId: 1 }
```

Comment:
maybe we should unify the variables naming convention on the bootstrap following some guideline
this line is using both snake_case and camelCase and should keep only one

Reply:
+1

Reply:
corrected